content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/100-connect-your-database.mdx

```diff
@@ -208,6 +208,64 @@ Adjust the connection URL to match your setup - see [Microsoft SQL Server connec
 
 </SwitchTech>
 
+<SwitchTech technologies={['*', 'cockroachdb']}>
+
+```prisma file=prisma/schema.prisma
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+```
+
+Note that the default schema created by `prisma init` uses PostgreSQL as the `provider`. For PlanetScale, you need to edit the `datasource` block to use the `cockroachdb` provider instead:
+
+```prisma file=prisma/schema.prisma highlight=2;edit
+datasource db {
+  provider = "cockroachdb"
+  url      = env("DATABASE_URL")
+}
+```
+
+You will also need to add the `cockroachdb` preview feature flag to the `generator` block of your `schema.prisma` file:
+
+```prisma file=schema.prisma highlight=3;add
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["cockroachdb"]
+}
+
+datasource db {
+  provider = "cockroachdb"
+  url      = env("DATABASE_URL")
+}
+```
+
+The `url` is [set via an environment variable](/guides/development-environment/environment-variables) which is defined in `.env`. For a CockroachDB database, the [connection URL](/reference/database-reference/connection-urls) looks similar to this:
+
+```bash file=.env
+DATABASE_URL="postgresql://myusername:mypassword@free-tier5.gcp-europe-west1.cockroachlabs.cloud:26257/defaultdb?sslmode=verify-full&sslrootcert=$HOME/.postgresql/root.crt&options=--mycluster
```

Comment:
This is a Cockroach Labs/Cloud connection string. For a local installation it might look like `postgresql://root@localhost:26257?sslmode=disable` (source: https://www.cockroachlabs.com/docs/v21.2/hello-world-node-postgres?filters=local)

Reply:
I've now added the local installation version of the connection string. Also CockroachDB Dedicated (can remove again if that's not relevant/too much...).